lib/Maintenance/Tasks/CleanupBrickTablesTask.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Maintenance\Tasks;
+
+use Pimcore\Db;
+use Pimcore\Maintenance\TaskInterface;
+use Pimcore\Model\DataObject\ClassDefinition;
+use Pimcore\Model\DataObject\Objectbrick\Definition;
+use Psr\Log\LoggerInterface;
+
+final class CleanupBrickTablesTask implements TaskInterface
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @param LoggerInterface $logger
+     */
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function execute()
+    {
+        $db = Db::get();
```

Comment:
inject via DI

Reply:
@weisswurstkanone no, please not, it will break the installer ;) Thanks!